RELEASES.md

```diff
@@ -1,3 +1,121 @@
+Version 1.39.0 (2019-11-07)
+===========================
+
+Language
+--------
+- [You can now create `async` functions and blocks with `async fn`, `async move {}`, and
+  `async {}` respectively, and you can now call `.await` on async expressions.][63209]
+- [You can now use `cfg` attributes on function parameters.][64010] e.g.
```

Comment:
This also applies to lint attributes, which I think is worth mentioning as well here